las/client.py

```diff
@@ -861,9 +874,32 @@ def update_transition_execution(
             'status': status,
             'output': output,
             'error': error,
+            'startTime': f'{start_time}' if start_time else start_time,
```

Comment:
Would else None be more readable?
```suggestion
            'startTime': f'{start_time}' if start_time else None,
```